### PR TITLE
New data loader - Opensearch 

### PR DESCRIPTION
**Pull request by chnsagitchen**
# Description

Due to the ElasticSearch client compatibility described: https://opensearch.org/docs/latest/clients/index/, higher version of the ES client, e.g. 8.9, is not recommended to work with latest OpenSearch. Also considering the ES and OpenSearch are separated, so it's better to have specific data loader with OpenSearch specific client for OpenSearch.

Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [ ] New Loader

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] I stared at the code and made sure it makes sense
- [ ] I tested the new loader from the download_loader with my local OpenSearch cluster

# Suggested Checklist:

- [ ] I have added a library.json file if a new loader/tool was added
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I ran `make format; make lint` to appease the lint gods

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -21,6 +21,7 @@ html2text = "*"
 psutil = "*"
 retrying = "*"
 pyaml = "^23.9.7"
+opensearch-py = "^2.4.2"
```
Comment (EmanuelCampos) [inline pyproject.toml:24]: you don't need this here, just keep the one on the opensearch folder

**Review by EmanuelCampos — COMMENTED** [review]
did a comment, a small thing

and can you run make format; make lint and make sure the tests passes?

File: pyproject.toml (hunk 1 above)
Comment (chnsagitchen) [reply pyproject.toml]: thanks @EmanuelCampos ! remove this.

**Comment by chnsagitchen** [comment]
> did a comment, a small thing
> 
> and can you run make format; make lint and make sure the tests passes?

@EmanuelCampos format the code and ran the tests locally successfully.

**Review by EmanuelCampos — APPROVED** [review]
Nice one, thanks for the contribution!

**Pull request merged**